app/src/main/java/org/kinecosystem/kinit/viewmodel/spend/TransferActivityModel.kt

```diff
@@ -41,10 +41,10 @@ class TransferActivityModel(private val app: EcosystemApp, var transferActions:
         if (app.transferData != null) {
             val started = transferManager.startTransferRequestActivity(activity, app.identifier, app.transferData.launchActivityFullPath)
             if (!started) {
-                transferActions?.onConnectionError()
+                transferActions?.onStartConnectingError()
             }
         } else {
-            transferActions?.onConnectionError()
+            transferActions?.onStartConnectingError()
```

Comment:
in this case, best to show the generic error message (this is not supposed to happen)

Reply:
ok